Dockerfile

```diff
@@ -12,6 +12,7 @@ ENV DATABASE_URL mysql2://root:root@mysql/signon
 ENV GOVUK_APP_NAME signon
 ENV PORT 3016
 ENV TEST_DATABASE_URL mysql2://root:root@mysql/signon_test
+ENV ASSETS_PREFIX /assets/signon
```

Comment:
Do we want to override this in the Dockerfile? (as opposed to setting it in Terraform?)

Reply:
We need it in both places. 

In the Dockerfile, it is needed so that the assets are generated in the right place. It won't work otherwise, even if you copy the assets later on in the right prefixpath in S3. I spent a lot of time troubleshooting that.

Reply:
Oh, I see. Didn't realise it was used at build-time as well. Thanks!